conda_build_prepare/git_helpers.py

```diff
@@ -145,10 +145,11 @@ def _call_custom_git_cmd(git_repo, cmd_string, check=True, quiet=False):
     return stdout.strip()
 
 
-def get_latest_describe_tag(git_repo):
-    # Find the `git describe` tag having any version-like part
```

Comment:
I think you are correct in that this comment was left over or incorrectly copied from somewhere else.